prow/pjutil/pjutil_test.go

```diff
@@ -1211,6 +1212,28 @@ func TestSpecFromJobBase(t *testing.T) {
 				return nil
 			},
 		},
+		{
+			name: "Verify DecorateExtraRefs",
+			jobBase: config.JobBase{
+				UtilityConfig: config.UtilityConfig{
+					DecorationConfig: &prowapi.DecorationConfig{
+						BloblessFetch: &trueVar,
+					},
+					ExtraRefs: []prowapi.Refs{{
+						Org: "test-org",
+						BloblessFetch: false,
+					}},
+				},
+			},
+			verify: func(pj prowapi.ProwJobSpec) error {
+				for _, er := range pj.ExtraRefs {
```

Comment:
Nit: I would prefer `extraRef` over `er` for readability. The `er` looks too close to `err` for me. Perhaps even `defaultedExtraRef` is another good candidate, because you're expecting the BloblessFetch in the DecorationConfig to override the false value in ExtraRefs.

Reply:
Agree, er is almost err. Hope just 'r' is not confusing. I like to keep loop variables short to maintain readability, and it's Go style.

Also, create a new slice just in case it's not safe to modify the one from the config.JobBase.